context/app/static/js/components/workspaces/JobLink.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+
+import { AppContext } from 'js/components/Providers';
+import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
+import { startJob } from './utils';
+import { LinkButton } from './style';
+
+function JobLink({ workspace, job, children }) {
+  const { workspacesEndpoint, workspacesToken } = useContext(AppContext);
+
+  function createHandleStart(workspaceId) {
+    async function handleStart() {
+      startJob({ workspaceId, workspacesEndpoint, workspacesToken });
+      // TODO: Open new tab
+      // eslint-disable-next-line no-alert
+      alert('TODO: Open a new tab that will poll until the job is started.');
+    }
+    return handleStart;
+  }
+
+  if (job.allowNew) {
+    return (
+      <OutboundIconLink>
+        <LinkButton onClick={createHandleStart(workspace.id)}>{children}</LinkButton>
+      </OutboundIconLink>
+    );
+  }
```

Comment:
```diff
     return (
-      <OutboundIconLink>
-        <LinkButton onClick={createHandleStart(workspace.id)}>{children}</LinkButton>
+      <OutboundIconLink component="button" onClick={createHandleStart(workspace.id)}>
+        {children}
       </OutboundIconLink>
     );
```
Baseline of text changes, along with weight. Fixing the weight is easy, but not sure how to align baseline.
<img width="220" alt="Screen Shot 2022-08-10 at 9 44 50 AM" src="https://user-images.githubusercontent.com/730388/183916960-6de0307f-784f-4efe-ba21-91d9a8794694.png">

--------

```diff
     return (
-      <OutboundIconLink>
-        <LinkButton onClick={createHandleStart(workspace.id)}>{children}</LinkButton>
+      <OutboundIconLink component={LinkButton} onClick={createHandleStart(workspace.id)}>
+        {children}
       </OutboundIconLink>
     );
```
Now the text baseline is good, but the icon baseline has moved.

<img width="232" alt="Screen Shot 2022-08-10 at 10 02 39 AM" src="https://user-images.githubusercontent.com/730388/183921650-e5f8b713-cc76-4148-94b0-2e9c78ffabcf.png">